packages/EasyCore/src/Bridge/Symfony/Resources/config/api_platform/filters.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+use EonX\EasyCore\Bridge\Symfony\ApiPlatform\Filter\VirtualSearchFilter;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+use function Symfony\Component\DependencyInjection\Loader\Configurator\service;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+
+    $services->set('eonx.api_platform.doctrine.orm.virtual_search_filter', VirtualSearchFilter::class)
+        ->autoconfigure(false)
+        ->autowire(false)
+        ->args([
+            service('doctrine'),
```

Comment:
`service()` function cannot be used yet as this monorepo supports Symfony 4.4
Please use `ref()` instead for now.

A seperate PR will be used to prepare the entire monorepo to either drop support for Symfony 4.4, or implement the required BC compatibility code, thanks.

Reply:
Done.